dataprofiler/profilers/profile_builder.py

```diff
@@ -788,7 +858,31 @@ def save(self, filepath=None):
         :type filepath: String
         :return: None
         """
-        raise NotImplementedError()
+        # Set Default filepath
+        if filepath is None:
+            filepath = "profile-{}.pkl".format(
+                datetime.now().strftime("%d-%b-%Y-%H:%M:%S.%f"))
```

Comment:
Could also add the dataset name maybe?

Reply:
Matching structured save / load. Plus a profile could be multiple datasets.

Reply:
FYI @lettergram